packages/input-select/src/Select.js

```diff
@@ -70,6 +70,7 @@ class Select extends React.Component {
     this.props.onChange(item.value);
   };
 
+  // eslint-disable-next-line
```

Comment:
It's `consistent-return` that is throwing an error, right?
Let's disable just that.